run_configs/_cron/build_all_gfortran.sh

```diff
@@ -0,0 +1,10 @@
+set -e
```

Comment:
I also like to set `-u` which treats dereferencing an undefined macro as an error. This is probably not relevant here as no macros are used. Just thought I'd mention it though.

Reply:
Thanks, I didn't know about that!